bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoDevice.java

```diff
@@ -264,30 +465,293 @@ else if (p_protocol_read.getProperty("messageType").equalsIgnoreCase("CEN Basic
+                                logger.debug("T_ControlMode : {}", l_item.getName());
+                                eventPublisher.postUpdate(l_item.getName(),
+                                        DecimalType.valueOf(p_protocol_read.getProperty("what")));
+                            }
+                            break;
+
+                        case 102:
+                            // Status Main Unit Remote
+                            if (p_protocol_read.getProperty("hStatus").equalsIgnoreCase("3")) {
+                                logger.debug("T_ControlRemote : {}", l_item.getName());
+                                eventPublisher.postUpdate(l_item.getName(),
+                                        DecimalType.valueOf(p_protocol_read.getProperty("what")));
+                            }
+                            break;
                     }
                 }
-            } else if (l_item instanceof NumberItem) {
-                logger.debug("Gateway [" + m_gateway_id + "], RECEIVED EVENT FOR NumberItem [" + l_item.getName()
-                        + "], TRANSLATE TO OPENHAB BUS EVENT");
+            } else if (l_item instanceof ContactItem) {
+                logger.debug("Gateway [{}], RECEIVED EVENT FOR ContactItem [{}], TRANSLATE TO OPENHAB BUS EVENT",
+                        m_gateway_id, l_item.getName());
 
                 // THERMOREGULATION
                 if (p_protocol_read.getProperty("messageType").equalsIgnoreCase("thermoregulation")) {
+                    switch (Integer.parseInt(l_binding_config.what)) {
+                        case 20:
+                            // Status Actuators
+                            if (p_protocol_read.getProperty("what").equalsIgnoreCase(l_binding_config.what)) {
+                                int state = -1;
+                                if (p_protocol_read.getProperty("messageDescription").equalsIgnoreCase("Heating OFF")) {
+                                    state = 0;
+                                    eventPublisher.postUpdate(l_item.getName(),
+                                            (state == 1) ? OpenClosedType.OPEN : OpenClosedType.CLOSED);
+                                } else if (p_protocol_read.getProperty("messageDescription")
+                                        .equalsIgnoreCase("Heating ON")) {
+                                    state = 1;
+                                    eventPublisher.postUpdate(l_item.getName(),
+                                            (state == 1) ? OpenClosedType.OPEN : OpenClosedType.CLOSED);
+                                }
+                                logger.debug("T_Relay : {} / {}", l_item.getName(), state);
+                            }
+                            break;
+                        case 102:
+                            // Status Main Unit Remote
+                            if (p_protocol_read.getProperty("hStatus").equalsIgnoreCase("3")) {
```

Comment:
No point in using `equalsIgnoreCase()` here

Reply:
OK
see commit https://github.com/openhab/openhab1-addons/pull/5486/commits/c1c8f3a86ba8641c7333467de21a0ba489322073